### PR TITLE
Add Sendy node documentation

### PR DESCRIPTION
**Pull request by harshil1712**

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/Sendy/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+The Start node exists by default when you create a new workflow.
+
+
+### 2. Sendy node (add: subscriber)
+
+This node will add a subscriber to a list in Sendy.
+
+1. First of all, you'll have to enter credentials for the Sendy node. You can find out how to do that [here](../../../credentials/Sendy/README.md).
+2. Enter the subscriber's email address in the ***Email*** field.
+3. Enter the list id in the ***List ID*** field.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Sendy/README.md:49]: Is this a prerequisite? If yes, it should be mentioned above. Also, how can one obtain a list ID?

File: docs/nodes/nodes-library/nodes/Sendy/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+2. Select 'Campaign' from the ***Resource*** dropdown list.
+3. Enter the name of the sender in the ***From Name*** field.
+4. Enter the email address of the sender in the ***From Email*** field.
+5. Enter the email address where the subscribers can reply to in the ***Reply To*** field.
+6. Enter the title of the campaign in the ***Title*** field.
+7. Enter the subject of the email in the ***Subject*** field.
+8. Enter the HTML snippet of the email campaign in the ***HTML Text*** field.
+9. Toggle ***Send Campaign*** to true.
+10. Click on the ***Add Field*** button and select 'List IDs'.
+11. Enter the list id in the ***List IDs*** field.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Sendy/README.md:72]: Is it the same list ID from the previous node? If yes, can you please use an expression here and update the workflow and the screenshot accordingly?

File: docs/nodes/credentials/Sendy/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Prerequisites
+
+Host a [Sendy](https://sendy.co/get-started) application.
+
+## Using API Key
+
+1. Access your Sendy application.
+2. Click on the username on the top right corner and select 'Settings'.
+3. Copy the API key displayed under the ***Your API key*** section.
+4. Use this API key and the installation URL with your Sendy node credentials in n8n.
```
Comment (tanay1337) [inline docs/nodes/credentials/Sendy/README.md:20]: Screenshot 2020-10-22 at 09 22 18

**Review by tanay1337 — CHANGES_REQUESTED** [review]

File: docs/nodes/credentials/Sendy/README.md (hunk 3 above)
Comment (harshil1712) [reply docs/nodes/credentials/Sendy/README.md:20]: @tanay1337 this is how it is displayed in Sendy. I am not sure if I am missing something. 

Screenshot 2020-10-22 at 3 54 51 PM

Comment (tanay1337) [reply docs/nodes/credentials/Sendy/README.md:20]: Ah okay, never mind then!

**Pull request merged**